xml/System.Net.Http/HttpClient.xml

```diff
@@ -363,9 +363,36 @@ public class GoodController : ApiController
         <ReturnType>System.Net.IWebProxy</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the global Http proxy.</summary>
+        <value>A proxy used by every call that instantiates a <see cref="T:System.Net.Http.HttpWebRequest" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+The <xref:System.Net.Http.HttpClient.DefaultProxy%2A> property gets or sets the global proxy. The <xref:System.Net.Http.HttpClient.DefaultProxy%2A> property determines the default proxy that all <xref:System.Net.Http.HttpClient> instances use if no proxy is set explicitly in the <xref:System.Net.Http.HttpClientHandler> passed through its constructor. 
+
+If the <xref:System.Net.Http.HttpClient.DefaultProxy%2A> property is overridden, all subsequent instances of the <xref:System.Net.Http.HttpClient> class that does not define its own Proxy in <xref:System.Net.Http.HttpClientHandler> will be using the new defined Proxy in <xref:System.Net.Http.HttpClient.DefaultProxy%2A>.
```

Comment:
I think this paragraph is redundant with the previous one. If you keep it, there is some rewording that needs to be done.

Reply:
Since this deals with overrides by a derived class, it's not redundant. But is this actually accurate? You wouldn't expect that a derived class would override not only its behavior, but also the behavior of its base class.

Reply:
I was not trying to talk about class overriding. I meant changing the value of the static property. I will reword that to make it more clear.